src/index.js

```diff
@@ -26,13 +25,18 @@ export default function(){
   }
 
   return {
+    setLoginToken: (token) => (
```

Comment:
this seems a bit strange. Can you explain a bit?

Reply:
I think his use case makes use of this to login the user without the `getLoginToken`

can you add this to the README @arthurnobrega ?

Reply:
Hey @guilhermedecampo and @grsabreu, thanks for the review!

This piece of code I added to be able to save the token on a database until it is expired, because I was receiving a lot of "Too many requests" from NuBank when I was testing.

So, on my use case I call once the getLoginToken, save it to a database, and the next time I just use setLoginToken to initialize nubank-api. You can see the project here: https://github.com/arthurnobrega/nubank-ofx

Reply:
I think you could just start a brand new one with the client_secret update.